src/reducers/profileReducer.js

```diff
@@ -26,9 +39,10 @@ function profileReducer(state = initialState, action) {
         return {
             ...state,
             profiles: action.data,
+            paginatedProfiles : applyPagination(1,action.data),
             activeProfiles: action.data.filter(profile =>
                 activeProfile.includes(profile.status)),
-            allButRejectedProfiles: action.data.filter(profile => profile.status != "Rejected"),
+            allButRejectedProfiles: getAllButRejectedProfiles(action.data),
             stats:{
               pieChart:{                
                 offerProfilesCount: action.data.filter(profile => profile.status == "Offer Made").length,
```

Comment:
Please use Constant for the "offer made" string

Reply:
Constant created